### DISCUSSION TITLE
Is there a way to use a set of svg paths for the symbol in Plot.dot?

### DISCUSSION THREAD

**Discussion by fernhaven** [observablehq/plot#2238] [Q&A] [answered]
Is it possible to use svg paths to display custom symbols with Plot.dot?

**Comment by Fil** [comment, marked as answer]
You can see how to create symbols via a series of "turtle" commands in all these files:
https://github.com/d3/d3-shape/tree/main/src/symbol

The commands are documented here
https://observablehq.com/@d3/d3-path

It's a subset of the SVG path spec. I don't recall if someone wrote an adaptor that would read in a SVG path and would call the corresponding commands — but that should be possible.
